.gitignore

```diff
@@ -0,0 +1,93 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
Is it possible to remove this .gitignore? I think a few of the references are configuration specific. We might need a `.gitignore` to ignore files generated by the library but I think that most of the other cases can be fixed with a global gitignore.

See [`.gitignore` from Django](https://github.com/django/django/blob/72be3152d77cd87c90f70a0c365ae7752804a86c/.gitignore) and the [github doc about the global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).